WordPress/gutenberg
# Draggable: Convert component to TypeScript

## What?
Refactor the `Draggable` component to TypeScript.

## Why?
Part of the @wordpress/components's TypeScript migration (https://github.com/WordPress/gutenberg/issues/35744).

## How?
- Refactored the component TypeScript
- Refactored storybook to typescript and controls

## Testing Instructions
1. Run npm run dev and ensure no typing errors
2. Test the Draggable in the storybook.
3. Test the various use of Draggable within the block/site editors in Gutenberg still works
